### PR TITLE
fw-4383, person APIs

### PR DESCRIPTION
**Pull request by sarahfirstvoices**
### Description of Changes
* new permissions for editing core language data (see User Roles for fv-be in team wiki)
* person read API 
* bonus changes: person create, update, and delete APIs

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- [ ] Admin Panel has been updated if models have been added or modified
- [ ] Migrations have been updated and committed if applicable

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/permissions/predicates/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 from .base import *  # noqa F401
+from .edit2 import *  # noqa F401
```
Comment (sarahfirstvoices) [inline firstvoices/backend/permissions/predicates/__init__.py:2]: I named it this just to avoid an annoying merge with the categories PR. I'll rename the module to `edit` when i merge them

File: firstvoices/backend/serializers/base_serializers.py [hunk 2]
```diff
@@ -7,15 +7,13 @@
 base_id_fields = ("id", "title", "url")
```
Comment (sarahfirstvoices) [inline firstvoices/backend/serializers/base_serializers.py:9]: Rough diff as usual, sorry. I extracted the site content url field to a mixin so it could be used for models that don't have a title. (Like Person.)

File: firstvoices/backend/tests/test_apis/base_api_test.py [hunk 3]
```diff
@@ -459,11 +459,7 @@ class DestroyApiTestMixin:
 
     @pytest.mark.django_db
```
Comment (sarahfirstvoices) [inline firstvoices/backend/tests/test_apis/base_api_test.py:460]: I changed the base delete tests to use a superadmin, since not all models are editable by a language admin

File: firstvoices/backend/urls.py [hunk 4]
```diff
@@ -9,34 +9,36 @@
...
 # app-level APIs
 ROUTER = DefaultRouter(trailing_slash=True)
-ROUTER.register(r"user", UserViewSet, basename=r"user")
-ROUTER.register(r"search", SearchViewSet, basename="search")
-ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="partofspeech")
 ROUTER.register(r"my-sites", MySitesViewSet, basename="my-sites")
+ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="partofspeech")
+ROUTER.register(r"search", SearchViewSet, basename="search")
+ROUTER.register(r"user", UserViewSet, basename=r"user")
 ROUTER.register(r"sites", SiteViewSet, basename="site")
```
Comment (sarahfirstvoices) [inline firstvoices/backend/urls.py:25]: I alphabetized these, except for leaving "sites" at the bottom. Open to other opinions; I was just finding it hard to find things in the long list. (Exciting that it is now long enough that this is a problem!)

File: firstvoices/backend/views/category_views.py [hunk 5]
```diff
@@ -96,7 +96,6 @@
         responses={
             204: OpenApiResponse(
                 description=doc_strings.success_204_deleted,
-                response=CategoryDetailSerializer,
```
Comment (sarahfirstvoices) [inline firstvoices/backend/views/category_views.py:99]: Unrelated minor fix-- there is no content for the delete response

File: firstvoices/backend/urls.py (hunk 4 above)
Comment (bronson-c) [reply firstvoices/backend/urls.py:26]: +1 for alphabetization

**Comment by bronson-c** [comment]
Oh I just thought of one minor change to add. The person api link should be added to the site serializer as well

File: firstvoices/backend/permissions/predicates/__init__.py (hunk 1 above)
Comment (sarahfirstvoices) [reply firstvoices/backend/permissions/predicates/__init__.py]: this is renamed now

**Review by sahil97 — APPROVED** [review]
